File: mifosng-android/src/main/java/com/mifos/utils/MifosResponseHandler.java
package com.mifos.utils;

/**
 * Created by Rajan Maurya on 08/07/16.
 */
public class MifosResponseHandler {

    public MifosResponseHandler() {

    }

    public static String getResponse() {
        switch (PrefManager.getUserStatus()) {

            case 0:
                return "created successfully";
            case 1:
                return "Saved into DB Successfully";

            default:
                return "";
        }

    }
}

File: mifosng-android/src/main/java/com/mifos/utils/MifosResponseHandler.kt
package com.mifos.utils

/**
 * Created by Rajan Maurya on 08/07/16.
 */
object MifosResponseHandler {
    fun getResponse(userStatus: Boolean): String {
        return when (userStatus) {
            true -> "Saved into DB Successfully"
            false -> "created successfully"
        }
    }
}